docs/articles_en/documentation/openvino_ir/operation_sets/operations_specifications/detection/DeformablePSROIPooling_1.rst

```diff
@@ -21,7 +21,6 @@ If only two inputs are provided, position sensitive pooling with regular ROI bin
 If third input is provided, each bin position is transformed by adding corresponding offset to the bin left top corner coordinates. Third input values are usually calculated by regular position sensitive pooling layer, so non-deformable mode (DeformablePSROIPooling with two inputs).
 The ROI coordinates are specified as five element tuples: ``[batch_id, x_1, y_1, x_2, y_2]`` in absolute values.
 
-This operation is compatible with `Apache MXNet DeformablePSROIPooling <https://mxnet.apache.org/versions/1.7.0/api/python/docs/api/contrib/symbol/index.html#mxnet.contrib.symbol.DeformablePSROIPooling>`__ cases where ``group_size`` is equal to ``pooled_size``.
```

Comment:
This relates to OpenVINO supporting the framework indirectly. We could consult it with developers.

Reply:
i agree with Karol, let's leave this reference, link is working and it might useful to know

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/22509/commits/dcc29c2247b8ff64a8d80be7591eff02ce87d58f